spec/parser/ruby/ruby_parser_spec.rb

```diff
@@ -283,6 +283,54 @@ def hello; end
       end
     end
 
+    it "correcly parses several formats of percent arrays in constant declarations" do
```

Comment:
You can rewrite this single test as a series of `it()` blocks (move the `it` call into the `str_a.each` block). That way you don't have to do your own failure output management at the bottom-- rspec can handle this automatically.